webapp/src/marked.tsx

```diff
@@ -303,6 +303,40 @@ export class MarkedContent extends data.Component<MarkedContentProps, MarkedCont
             })
     }
 
+    private renderBullets(content: HTMLElement) {
+        const bulletRegex = /^:([^:]+):/i;
+        pxt.Util.toArray(content.querySelectorAll("li"))
+            .forEach((li: HTMLElement) => {
+                const match = li.innerHTML.match(bulletRegex);
+                if (match?.[1]) {
+                    const p = document.createElement("p");
+                    p.innerHTML = li.innerHTML;
+                    li.innerHTML = "";
+                    li.appendChild(p);
+                }
+            })
+
+        pxt.Util.toArray(content.querySelectorAll("li > p"))
+            .forEach((p: HTMLElement) => {
+                const match = p.innerText.match(bulletRegex);
+                if (match?.[1]) {
+                    p.innerHTML = p.innerHTML.replace(bulletRegex, "");
```

Comment:
you can probably just run this on the first text node child to get around this warning